src/unix/osd.c

```diff
@@ -48,6 +48,7 @@
 #include "fi.h"
 #include "fi_osd.h"
 #include "fi_file.h"
+#include "fi_util.h"
```

Comment:
What is the purpose of including this file? Does this file work with some stuff from `fi_util.h` ?

Reply:
It's in the commit message:
    osd: added fi_util.h header
    
    The header contains prototypes for ofi_shm_map() and ofi_shm_unmap().

Reply:
Warning is:

    CC       src/unix/src_libfabric_la-osd.lo
    src/unix/osd.c:85:5: warning: no previous prototype for ‘ofi_shm_map’ [-Wmissing-prototypes]
     int ofi_shm_map(struct util_shm *shm, const char *name, size_t size,
         ^~~~~~~~~~~
    src/unix/osd.c:163:5: warning: no previous prototype for ‘ofi_shm_unmap’ [-Wmissing-prototypes]
     int ofi_shm_unmap(struct util_shm* shm)
         ^~~~~~~~~~~~~

Reply:
> It's in the commit message:
> osd: added fi_util.h header

Thanks for the explanation!